04/styles/mobile1.css

```diff
@@ -0,0 +1,81 @@
+/* hamburger icon */
+.menu__icon {
+  width: 40px;
+  height: 35px;
+  border: none;
+  border-top: 5px solid salmon;
+  background: transparent;
+  position: relative;
+  transition: .2s transform linear;
+}
+.menu__icon::before, .menu__icon::after {
+  content: '';
+  top: 0;
+  left: 0;
+  width: 100%;
+  border-top: 5px solid salmon;
+  transform: translateY(10px);
+  position: absolute;
+}
+.menu__icon::after {
+  transform: translateY(25px);
+  transition: .2s transform linear;
+}
+.menu__icon:focus {
+  transform: rotate(45deg) translateY(5px);
+  border: none;
+}
+.menu__icon:focus::after {
+  transform: rotate(-90deg) translateX(-10px);
+}
+
+/* jak odkomentuje te opcje zeby animowac HAMBURGERA to nie wyswietlaja mi sie podmenu, nie wiem jak to zrobic */
+/* nav ul {
+  display: none;
+} */
+
+button:focus + ul {
+   display: block;
+}
+/* hide checkbox */
+nav input {
+  display: none;
+}
+
+
+/* magic here */
+
+.nav__container ul {
+  list-style-type: none;
+  margin: 0;
+}
+
+/* co lepsze? */ /* ??? nav ul li CZY .nav__container li  ??? */
+.nav__container ul li {
```

Comment:
najlepiej nadać klasę dla li i to na podstawie jej stylować ;)

Reply:
OK